IaC/BareMetal/app/README.md

```diff
@@ -2,45 +2,80 @@
 
 ```bash
 
-# add secrets
-
-#### edit config.yaml for region / zone
```

Comment:
moved from main readme

I think we should simplify and only setup in-memory
That keeps us from having to create Cosmos DB and Log Analytics
We can have separate directories / instructions for setting those up

Reply:
For the new FluentBit + Azure Log Analytics, I built a simple app that writes logs for end-to-end validation.

I stole heavily from LodeRunner